### ISSUE TITLE
doubt about nail arts

### ISSUE THREAD

**Issue by LukianDagbo** [DistractionCrab/Charm-Crab#4]
The Nail Art always consume soul or only when you equip the charm? I was testing the amulets and I realized that the Nail Arts always consumes soul

**Comment by DistractionCrab** [comment]
If you're referring to Soul-Infused Blade, it will attempt to take soul each time (up to 99) to deal increased damage. The formula is (1 + S/99 * M) where S is the soul consumed and M is the maximum multiplier. This charm is meant to act as a soul-dump for nail/nail-art focused charm setups.

**Comment by LukianDagbo** [comment]
Talked about vanilla nails without charm.
I know Soul Infused Blade Cost Soul.
Nmg also does it cost soul?

**Comment by DistractionCrab** [comment]
I didn't get an alert for this, sorry for the late reply. I went back and realized I read the issue wrong. SIB should only take soul if it's equipped. I just fixed it. NMG should not be using soul, so if it is by itself (after this release) I'll need to fix something, but nothing in its code is taking it so it should be fine. I'll make sure to get this release pushed out soon, now that we've got the last of our icons finished.
